indexer/road_shields_parser.cpp

```diff
@@ -0,0 +1,308 @@
+#include "indexer/road_shields_parser.hpp"
+
+#include "indexer/feature.hpp"
+
+#include "base/string_utils.hpp"
+
+#include <unordered_map>
+#include <utility>
+
+namespace ftypes
+{
+class RoadShieldParser
```

Comment:
Please, put all these helper classes into anonymous namespace. Otherwise, it will be possible to reference them from other translation units.